format/FormatHDF5EigerNearlyNexus.py

```diff
@@ -50,7 +49,7 @@ def is_eiger_nearly_nexus_file(filename):
         if entry is not None:
             try:
                 return (
-                    numpy.string_("dectris eiger")
+                    np.string_("dectris eiger")
```

Comment:
That feeling when you see something a bit odd while reviewing code... wonder what this is used for...